### PR TITLE
feature/support-arrays

### PR DESCRIPTION
**Pull request by jonhindle**
### Context

> Does this issue have a Trello card?
https://trello.com/c/uiKtJkbe/943-feedback-table-not-working

> If this is an issue, do we have steps to reproduce?
feedback includes arrays

### Intent

> What changes are introduced by this PR that correspond to the above card?
all Array values are converted to strings

> Would this PR benefit from screenshots?
n/a

### Considerations

> Is there any additional information that would help when reviewing this PR?
n/a

> Are there any steps required when merging/deploying this PR?
n/a

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

**Review by andrewrlee — DISMISSED** [review]
LGTM

File: job/types.ts [hunk 1]
```diff
@@ -27,16 +29,17 @@ export class FeedbackItem {
...
+    const flatRows = flattenArrays(row)
+    this.date = flatRows.date
+    this.title = flatRows.title
+    this.contentType = flatRows.contentType
+    this.sentiment = flatRows.sentiment
+    this.comment = flatRows.comment
+    this.sessionId = flatRows.sessionId
+    this.establishment = flatRows.establishment
+    this.series = flatRows.series
+    this.categories = flatRows.categories
```
Comment (BarryGeeMOJ) [inline job/types.ts:42]: Do we need to be setting default / fall back values for each of these?

**Review by andrewrlee — APPROVED** [review]
LGTM

**Comment by psoleckimoj** [comment]
Closing, been open for 2 years

**Pull request closed without merging**